apps/token/src/routes/rewards/epoch-total-rewards/epoch-total-rewards.tsx

```diff
@@ -27,12 +27,12 @@ export const EpochRewards = () => {
           className="max-w-full overflow-auto"
           data-testid="epoch-rewards-total"
         >
-          {epochRewardSummaries.length === 0 ? (
+          {epochTotalRewardSummaries.length === 0 ? (
             <NoRewards />
           ) : (
             <>
-              {epochRewardSummaries.map((aggregatedEpochSummary) => (
-                <EpochTotalRewardsTable data={aggregatedEpochSummary} />
+              {epochTotalRewardSummaries.map((epochTotalSummary) => (
+                <EpochTotalRewardsTable data={epochTotalSummary} />
```

Comment:
suggestion (non-blocking): i think adding a `key={index}` (or any unique id from the epochTotalSummary) to this table would help get rid of the react warning when react complains about a components rendered in a loop not having a key prop.